website_portal_medical_prescription_order_line/controllers/main.py

```diff
@@ -28,6 +28,21 @@ def my_medical(self, **kw):
             ('patient_id.parent_id', 'child_of', [partner_id.id]),
         ])
 
+        url = str(request.httprequest)
```

Comment:
Here are the errors from `request.httprequest.getlist("patients")`:

![screen shot 2017-04-03 at 10 18 36 am](https://cloud.githubusercontent.com/assets/18649466/24621663/208eb8b8-1857-11e7-982c-f409da8a576f.png)

![screen shot 2017-04-03 at 10 18 44 am](https://cloud.githubusercontent.com/assets/18649466/24621671/253a160a-1857-11e7-8931-265a9256dfaf.png)